static/locales/en.client.json

```
@@ -1,18 +1,763 @@
+    "NotFoundMainText": "The requested page could not be found.{{separator}}Please check the URL and try again.",
+    "ContactSupport": "Contact support",
+    "SomethingWentWrong": "Something went wrong",
+    "ErrorHappened_message": "There was an error: {{message}}",
+    "ErrorHappened_unknown": "There was an unknown error."
+  },
+  "FieldConfig": {
+    "ColumnLabel": "COLUMN LABEL AND ID",
+    "ColumnOptionsLimited": "Column options are limited in summary tables.",
+    "ColumnType_formula_one": "Formula Column",
+    "ColumnType_data_one": "Data Column",
+    "ColumnType_empty_one": "Empty Column",
+    "ColumnType_formula_other": "Formula Columns",
+    "ColumnType_data_other": "Data Columns",
+    "ColumnType_empty_other": "Empty Columns",
+    "ColumnType_mixed_other": "Mixed Behavior",
+    "ConvertColumn_formula": "Clear and make into formula",
+    "ConvertColumn_data": "Convert column to data",
+    "ConvertColumn_triggerformula": "Convert to trigger formula",
+    "ClearAndReset": "Clear and reset",
+    "EnterFormula": "Enter formula",
+    "ColumnBehavior": "COLUMN BEHAVIOR",
+    "SetFormula": "Set formula",
+    "SetTriggerFormula": "Set trigger formula",
+    "MakeIntoDataColumn": "Make into data column",
+    "TriggerFormula": "TRIGGER FORMULA"
+  },
+  "ExampleInfo": {
+    "Title_CRM": "Lightweight CRM",
+    "WelcomeTitle_CRM": "Welcome to the Lightweight CRM template",
+    "WelcomeText_CRM": "Check out our related tutorial for how to link data, and create high-productivity layouts.",
+    "WelcomeTutorialName_CRM": "Tutorial: Create a CRM",
+    "Title_investmentResearch": "Investment Research",
+    "WelcomeTitle_investmentResearch": "Welcome to the Investment Research template",
+    "WelcomeText_investmentResearch": "Check out our related tutorial to learn how to create summary tables and charts, and to link charts dynamically.",
+    "WelcomeTutorialName_investmentResearch": "Tutorial: Analyze & Visualize",
+    "Title_afterschool": "Afterschool Program",
+    "WelcomeTitle_afterschool": "Welcome to the Afterschool Program template",
+    "WelcomeText_afterschool": "Check out our related tutorial for how to model business data, use formulas, and manage complexity.",
+    "WelcomeTutorialName_afterschool": "Tutorial: Manage Business Data"
+  },
+  "FieldMenus": {
+    "UsingSettings_common": "Using common settings",
+    "UsingSettings_separate": "Using separate settings",
+    "Settings_useseparate": "Use separate settings",
+    "Settings_savecommon": "Save as common settings",
+    "Settings_revertcommon": "Revert to common settings"
+  },
+  "FilterBar": {
+    "AddFilter": "Add Filter"
+  },
+  "GridOptions": {
+    "GridOptions": "Grid Options",
+    "VerticalGridlines": "Vertical Gridlines",
+    "HorizontalGridlines": "Horizontal Gridlines",
+    "ZebraStripes": "Zebra Stripes"
+  },
+  "GridViewMenus": {
+    "AddColumn": "Add Column",
+    "ShowColumn": "Show column {{label}}",
```

```suggestion
    "ShowColumn": "Show column {{- label}}",
```

(For column labels that have special characters.)